ResuMe/urls.py

```diff
@@ -14,15 +14,16 @@
     2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
 """
 from django.contrib import admin
-from django.urls import path
-from posts import views
+from django.urls import path, include
 from django.conf import settings
 from django.conf.urls.static import static
 
 
 urlpatterns = [
     path('admin/', admin.site.urls),
-    path('', views.posts, name='posts')
+    path('', include('posts.urls')),
```

Comment:
I think we should add a path name, something like:
 path('users/', include('users.urls'),name='Profile')